week_4/consumer/consumer_test.go

```diff
@@ -0,0 +1,161 @@
+// +build integration
+
+package consumer
+
+import (
+	"log"
+	"testing"
+
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/pact-foundation/pact-go/dsl"
+)
+
+func TestCreateToDo(t *testing.T) {
+	// Create Pact connecting to local Daemon
+	pact := &dsl.Pact{
+		Consumer: "ToDoConsumer",
+		Provider: "ToDoService",
+		Host:     "localhost",
+	}
+	defer pact.Teardown()
```

Comment:
This approach starts new server for every test. Therefore, it's slow. Using sub-tests could improve execution time a lot.
https://github.com/xuanit/testing/blob/master/todo/consumer/consumer_test.go